src/api/getComments.ts

```diff
@@ -0,0 +1,9 @@
+import comments from './comments';
+
+import { Comment } from '../types/Comment';
```

Comment:
No need to use empty line here
```suggestion
import comments from './comments';

import { Comment } from '../types/Comment';
```